TestLogin/TestLogin/TestLogin/MainPage.xaml.cs

```diff
@@ -13,6 +13,22 @@ public partial class MainPage : ContentPage
         public MainPage()
         {
             InitializeComponent();
+            loginButton.Clicked += LoginButton_Clicked;
+            var username = new Entry {Text = "lahiru"};
+            var password = new Entry { Text = "lahiru"};
+        }
+        void LoginButton_Clicked(object sender, EventArgs e)
+        {
+            if (username.Text == "lahiru" && password.Text == "lahiru")
```

Comment:
Use try catch block for prevent the crashes when initialize the page 1

Reply:
Sure. I will Update